src/posthog-featureflags.ts

```diff
@@ -287,15 +290,15 @@ export class PostHogFeatureFlags {
         this._override_warning = false
 
         if (flags === false) {
-            this.instance.persistence.unregister(PERSISTENCE_OVERRIDE_FEATURE_FLAGS)
+            this.instance.persistence?.unregister(PERSISTENCE_OVERRIDE_FEATURE_FLAGS)
```

Comment:
I think it makes sense to play it safe overall, but e.g. here this will quietly fail (effectively) before  `persistence` is initialized, which seems dangerous. We should exit the whole function early instead, and have a `console.warning()`, so that the developer knows this is happening (considering you mentioned the dev environment use case specifically)

Reply:
Added it 👍